runtime/bajun/src/lib.rs

```diff
@@ -479,6 +480,29 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 
 impl parachain_info::Config for Runtime {}
 
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	items as Balance * 20 * BAJUN + (bytes as Balance) * 1_000 * MICRO_BAJUN
+}
+
+parameter_types! {
+	/// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
+	/// See [pallet_multisig::Config::DepositBase] for details
```

Comment:
Let's remove the second comment
```suggestion
	/// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
```